get_dependencies.py

```diff
@@ -5,9 +5,11 @@
 import requests
 
 
-def download(url, filename):
+def download(url):
+    response = requests.get(url, stream=True)
+    filename = response.headers.get('content-disposition')
+    filename = filename[filename.find("\"") + 1: filename.rfind("\"")]
```

Comment:
This is not implementing the fallback when the content-disposition header doesn't exist.
I suggest you use the `rfc6266` library to do this.

Reply:
@marco-c   i think the pr of mine which you closed recently implements your requirement, doesn't it?

Reply:
I don't think so, in your PR you're printing `File Not Found` when the content-disposition header doesn't exist and the `filename` variable is not defined in that case.

Reply:
OK. what should we store the value in the variable "filename" if the content-disposition header does not exist ? we can just add an else clause?

Reply:
Discussion at https://github.com/marco-c/autowebcompat/pull/48#discussion_r167229099.